### DISCUSSION TITLE
Lay Z Spa - 4 Wire Connection Query

### DISCUSSION THREAD

**Discussion by tricey27** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#785] [Q&A]
Hi All,

im just building up my PCB now and are a little unsure which holes on the PCB I solder my 4 wire version to. Can anyone advise please? Cheers

**Comment by visualapproach** [comment]
I tried to describe that in the manual. Can you be more specific about your thoughts?

**Reply by tricey27** [reply]
The loom within the hot tub is all white cables. I have installed the resistors and capacitors so I guess I use H1B and H2B. I’m just unsure of the 6 holes which four I would use and which pin on the harness is which? Thanks

**Reply by visualapproach** [reply]
You should use the marked holes. They are 'filled' squares on the pcb. The order is up to you to find out. It's the glory of DIY 😃 
Find out which wire is 5V and GND and then you only have 4 combinations. And you don't even have to guess. You can run the layzspa.local/inputs/ to see which wire is dsp tx and the rest can be figured out.
